raiden/network/transport/matrix/transport.py

```diff
@@ -1170,8 +1196,69 @@ def _handle_sync_messages(self, sync_messages: MatrixSyncMessages) -> bool:
 
         self._raiden_service.on_messages(all_messages)
 
+    def _handle_sync_messages(self, sync_messages: MatrixSyncMessages) -> bool:
+        """ Handle text messages sent to listening rooms """
+        if self._stop_event.ready():
+            return False
+
+        assert self._raiden_service is not None, "_raiden_service not set"
+
+        all_messages: List[Message] = list()
+        for room, room_messages in sync_messages:
+            for text in room_messages:
+                all_messages.extend(self._handle_text(room, text))
+
+        self._process_messages(all_messages)
         return len(all_messages) > 0
 
+    def _handle_web_rtc_messages(self, message_data: str, partner_address: Address) -> None:
+        if not self._stop_event.is_set():
+            messages = validate_and_parse_message(message_data, partner_address)
+            self._process_messages(messages)
+
+    def _handle_call_messages(self, sync_messages: MatrixSyncMessages) -> None:
+        """
+        This function handles incoming signalling messages (in matrix called 'call' events)
```

Comment:
I guess this comment is not correct any more

Reply:
It is. But I added an additional line to clarify